### ISSUE TITLE
Need better demo gif

### ISSUE THREAD

**Issue by Chris-N-K** [Chris-N-K/napari-time_series_plotter#12]
Still not sure what the best way would be to create a demo gif.
The current one's resolution is too low, but it is already at github's max size for file uploads...

Suggestions welcome!

**Comment by ch-n** [comment]
The new gif must show the different plotting modes as well!

**Comment by ch-n** [comment]
@rjlopez2 if you find some time and motivation your help would be highly appreciated!

**Comment by rjlopez2** [comment]
Absolutely 👍 
I had already some ideas on this front as well, I just need to have the time LOL but I think i need to think it trough fist.
I found a lot of inspiration in the many gif shown as "quick tutorial" by the many plugins from Robert Hasse.
Yet, I don't know what tools or compression they do the images but they look kinda of acceptable.
If we found a good compromise in quality and size of the gif(s), I was thinking on including additional gifs showing how to work the different plotting modes and paginated nicely with the current documentation (you did a nice work on this btw).
I will come with some ideas soon.
Cheers!
